src/components/Card.js

```diff
@@ -44,19 +47,16 @@ export default function Card({
           <span>Kategorien:</span>
         </ScreenReaderOnly>
         <CategoryWrapper role="list">
-          {categories?.map((category, index) =>
-            category ? (
-              <Category key={`${category}-${index}`}>{category}</Category>
-            ) : (
-              ''
-            )
+          {categories?.map(
+            (category, index) =>
+              category && (
```

Comment:
Is it possible to not have a `category`? It is a bit unusual to use conditional rendering within a `map` function.

Reply:
It didn't work when I first implemented it and I fixed it this way. I think sometimes there were errors because the rendering of the categories took too long, but this was a couple of user stories ago, so I'm not sure..

Reply:
NOW I remember: I did this, because if only 1 category is set, there were empty categories displayed on the cards 😅